depot-tools-git/PKGBUILD

```diff
@@ -11,7 +11,7 @@
 # Contributor: hack.augusto <hack.augusto@gmail.com>
```

Comment:
Care to add a comment noting yourself as a contributor?

Reply:
Ofc.